deploy.sh

```
@@ -0,0 +1,330 @@
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -e|--environment)
+            ENVIRONMENT="$2"
+            shift 2
+            ;;
+        -f|--values)
+            VALUES_FILE="$2"
+            shift 2
+            ;;
+        -n|--namespace)
+            NAMESPACE="$2"
```

Shellcheck (0.11.0)

[warning] NAMESPACE appears unused. Verify use (or export if used externally).

(SC2034)